skbio/tree/_tree.py

```diff
@@ -4306,9 +4310,9 @@ def _compare_topology(
         # unweighted (set difference)
         if not weighted:
             if symmetric:
-                result = sets1.symmetric_difference(sets2)
+                result = sets1 ^ sets2
```

Comment:
Many developers won't recognize what operation is being performed here and will need to google it. Can the explicit form be retained?

Reply:
Makes sense! Let me add a comment after each line of code.